sdk/python/pulumi_kubernetes/apiextensions/CustomResource.py

```diff
@@ -9,8 +9,90 @@
 from pulumi import ResourceOptions
 
 from .. import meta as _meta
-from .. import _utilities, _tables
+from .. import _utilities
+
+__all__ = ['CustomResourceArgs', 'CustomResource']
+
+@pulumi.input_type
+class CustomResourceArgs:
+    def __init__(__self__, *,
+                 api_version: pulumi.Input[str],
+                 kind: pulumi.Input[str],
+                 compat: Optional[pulumi.Input[str]] = None,
```

Comment:
What's the compat arg for? I don't see a corresponding docstring.

Reply:
Left over from attempt to codegen. Removed.